CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Upcoming]
 
+## [0.23.0] — 2020-07-09
```

Comment:
<!--
  1 failure:  The latest date i...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
  File: CHANGELOG.md;
  Line: 9;
-->
- :no_entry_sign: The latest date in the CHANGELOG is not today!


```suggestion
## [0.23.0] - 2020-07-09
```